app/assets/javascripts/components/common/calendar.jsx

```diff
@@ -23,6 +23,14 @@ const Calendar = createReactClass({
     updateCourse: PropTypes.func.isRequired
   },
 
+  statics: {
```

Comment:
If we won't be using any more static methods, it is better to go with `static getDerivedStateFromProps`, keeps the code concise.